boards/common/blxxxpill/Makefile.include

```diff
@@ -1,11 +1,15 @@
 INCLUDES += -I$(RIOTBOARD)/common/blxxxpill/include
 
-# For older versions of the DFU bootloader, set DFU_USB_ID to 1d50:6017 and
-# uncomment the DFU_FLAGS line
-DFU_USB_ID ?= 1eaf:0003
-#DFU_FLAGS ?= -s 0x08002000:leave
+ifeq (dfu-util,$(PROGRAMMER))
+  # For older versions of the DFU bootloader, set DFU_USB_ID to 1d50:6017
+  DFU_USB_ID ?= 1eaf:0003
+  DFU_ALT ?= 2
 
-STLINK_VERSION ?= 2
+  # Leave some space at the beginning of the flash for the bootloader
+  ROM_OFFSET ?= 0x2000
```

Comment:
Is this only needed for dfu?

Reply:
The CI fails otherwise. And in master, ROM_OFFSET is set for blxxxpill boards only if dfu-utll is used (no functional/configuration change here).